### PR TITLE
Improve 'effects' and 'instrument' parameters in browserClip

### PR DESCRIPTION
**Pull request by qchenevier**
See #108

### CODE REVIEW

**Comment by qchenevier** [comment]
I've discovered that some effects need to be started (= call the `.start()` method) to be effective. And some others don't have this method. I added a mapping function to start the effect if needed. See code [here](https://github.com/qchenevier/scribbletune/blob/master/src/browserClip.ts#L162-L168).

File: src/browserClip.ts [hunk 1]
```diff
@@ -151,8 +151,27 @@ module.exports = (params: ClipParams) => {
...
+    if (typeof eff === 'string') {
+      return new Tone[eff]();
+    } else {
+      return eff;
+    }
+  }
+
+  function startEffect(eff: any) {
+    if (typeof eff.start === 'function') {
+      return eff.start();
```
Comment (walmik) [inline src/browserClip.ts:164]: Is `start()` a new param in v14? Was looking at this page from 13.x as an example and couldnt find a `start` method https://tonejs.github.io/docs/13.8.25/AutoWah

File: src/browserClip.ts [hunk 2]
```diff
@@ -151,8 +151,27 @@ module.exports = (params: ClipParams) => {
...
+      return eff;
+    }
+  }
+
   if (params.effects) {
-    effects = params.effects.map((eff: any) => new Tone[eff]());
+    if (!Array.isArray(params.effects)) {
+      params.effects = [params.effects];
+    }
+    effects = params.effects.map(createEffect).map(startEffect);
```
Comment (walmik) [inline src/browserClip.ts:174]: Nice :)

**Comment by walmik** [comment]
Changes look great @qchenevier May i request you to do a minor version bump as part of this PR? Then all i have to do is just merge it and it ll go to CDN.js on it's own.

File: src/browserClip.ts (hunk 1 above)
Comment (qchenevier) [reply src/browserClip.ts:164]: Hmmm...  🤔 No, it's not new. `AutoWah` doesn't have this method. It seems that the `start()` method is present for some effects only. E.g:
- for `AutoFilter` the method is present [in v13](https://tonejs.github.io/docs/13.8.25/AutoFilter#start) and [in v14](https://tonejs.github.io/docs/14.7.11/AutoFilter#start)
- for `AutoWah` the method is absent for both versions.

Your comment reminded me that I tested only with Tone v14, but not with with Tone v13. So I did some tests with Tone v13 and it seems ok.

**Comment by qchenevier** [comment]
Done :) First time I use `npm version` so let me know if something is wrong.

**Pull request merged**